### ISSUE TITLE
Insert Images into Headers

### ISSUE THREAD

**Issue by SportStatisticsRSweet** [haozhu233/kableExtra#551]
I like the new feature of inserting [images into columns](https://cran.r-project.org/web/packages/kableExtra/vignettes/awesome_table_in_html.html#Insert_Images_into_Columns) but it would be super neat if this could also apply to headers? At present `row_spec(0, image = "kableExtra_sm.png")` throws an error and using custom css is very fiddly. It would be neat to have the option to add an image or logo, rather than using text in a header.

**Comment by haozhu233** [comment]
Sure, it would be great to have that feature. However, right now `row_spec` can only take single value input (same as how `column_spec` used to be`). Changing it from single value to vector requires quite some changes while I'm super busy recently with other things...  :P 

If you don't mind, I will try to add this feature in the next version.

**Comment by SportStatisticsRSweet** [comment]
Of course, no rush! I should have added, more a "suggestion" than an issue :-)

Thanks for considering as a feature in the next version.
